object-manipulation-2/src/pick.js

```diff
@@ -0,0 +1,12 @@
+/* exported pick */
+function pick(source, keys) {
+  var pickObject = {};
+  for (var keysIndex = 0; keysIndex < keys.length; keysIndex++) {
+    for (var property in source) {
+      if (keys[keysIndex] === property && source[property] !== undefined) {
+        pickObject[property] = source[property];
+      }
+    }
```

Comment:
This inner loop is not necessary, you can use `keys[keyIndex]` instead of your `property` variable.